app/controllers/work_controller.rb

```diff
@@ -193,75 +193,23 @@ def create
   end
 
   def update
-    @work = Work.find(params[:id].to_i)
-    id = @work.collection_id
-    @collection = @work.collection if @collection.nil?
-    #check the work transcription convention against the collection version
-    #if they're the same, don't update that attribute of the work
-    params_convention = params[:work][:transcription_conventions]
-    collection_convention = @work.collection.transcription_conventions
-
-    if params_convention == collection_convention
-      @work.attributes = work_params.except(:transcription_conventions)
-    else
-      @work.attributes = work_params
-    end
+    work = Work.find(params[:id])
+    result = Work::Update.call(work: work, work_params: work_params)
 
-    #if the slug field param is blank, set slug to original candidate
-    if work_params[:slug].blank?
-      @work.slug = @work.title.parameterize
-    end
+    @work = result.work
+    if result.success?
+      record_deed(@work, DeedType::WORK_ADDED, @work.owner) if result.original_collection_id != result.collection_id
```

Comment:
Doesn't work.update get called whenever the settings change?  We shouldn't be logging a WORK_ADDED deed for that.

Reply:
or maybe the question is when is a work "updated" vs "edit"ed?